products/windows.md

```diff
@@ -259,21 +282,24 @@ releases:
     latest: 6.2.9200
     link: https://learn.microsoft.com/lifecycle/products/windows-8
 
--   releaseCycle: "7 SP1"
+-   releaseCycle: "7-SP1"
```

Comment:
I preferred to use lowercase for all `releaseCycle` on iphone to avoid uppercase / lowercase confusion, maybe we could do the same here ?

Reply:
Thanks for the review and all the comments.

> By merging the two releases isn't there a risk that future changes makes us split it again, which will imply a breaking change of the API (similar to #3791) ?

No that will not happen. There are cycles defined by MS from the beginning that are equal to (E) and (W). I think we should label these that are only for (E) or (W) and remove the labels with labels (E) and (W). WDYT?

> I preferred to use lowercase for all `releaseCycle` on iphone to avoid uppercase / lowercase confusion, maybe we could do the same here ?

That's a good idea, will update...

Regarding the duplicates I will check again all entries and make the updates.

Reply:
> No that will not happen. There are cycles defined by MS from the beginning that are equal to (E) and (W). I think we should label these that are only for (E) or (W) and remove the labels with labels (E) and (W). WDYT?

Ok.

Reply:
Solved with 7a09b7d1df4efe13102d3ed2ad15420ea39799fd